src/components/MultiHopTrade/components/TradeInput/components/Claim/ClaimStatus.tsx

```diff
@@ -0,0 +1,157 @@
+import { CheckCircleIcon, WarningTwoIcon } from '@chakra-ui/icons'
+import { Button, CardBody, CardFooter, Center, Heading, Link, Stack } from '@chakra-ui/react'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import { useQueryClient } from '@tanstack/react-query'
+import { AnimatePresence } from 'framer-motion'
+import React, { useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { Amount } from 'components/Amount/Amount'
+import { CircularProgress } from 'components/CircularProgress/CircularProgress'
+import { SlideTransition } from 'components/SlideTransition'
+import { SlideTransitionY } from 'components/SlideTransitionY'
+import { getTxLink } from 'lib/getTxLink'
+import { fromBaseUnit } from 'lib/math'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import type { ClaimDetails } from './hooks/useArbitrumClaimsByStatus'
+import { ClaimRoutePaths } from './types'
+
+type ClaimStatusBodyProps = {
+  amountCryptoPrecision: string
+  description: string
+  icon: JSX.Element
+  status: TxStatus
+  symbol: string
+  title: string
+}
+
+export const ClaimStatusBody: React.FC<ClaimStatusBodyProps> = ({
+  amountCryptoPrecision,
+  description,
+  icon,
+  status,
+  symbol,
+  title,
+}) => {
+  return (
+    <SlideTransitionY key={status}>
+      <CardBody py={32}>
+        <Center flexDir='column' gap={4}>
+          {icon}
+          <Stack spacing={2} alignItems='center'>
+            <Heading as='h4'>{title}</Heading>
+            <Amount.Crypto
+              prefix={description}
+              value={amountCryptoPrecision}
+              symbol={symbol}
+              color='text.subtle'
+            />
+          </Stack>
+        </Center>
+      </CardBody>
+    </SlideTransitionY>
+  )
+}
+
+type ClaimStatusProps = {
+  activeClaim: ClaimDetails
+  claimTxHash: string
+  claimTxStatus: TxStatus
+}
+
+export const ClaimStatus: React.FC<ClaimStatusProps> = ({
+  activeClaim,
+  claimTxHash,
+  claimTxStatus,
+}) => {
+  const history = useHistory()
+  const translate = useTranslate()
+  const queryClient = useQueryClient()
+
+  const handleGoBack = useCallback(async () => {
+    await queryClient.invalidateQueries({
+      queryKey: ['claimStatus', { txid: activeClaim.tx.txid }],
+      refetchType: 'all',
+    })
+
+    history.push(ClaimRoutePaths.Select)
+  }, [activeClaim.tx.txid, history, queryClient])
+
+  const asset = useAppSelector(state => selectAssetById(state, activeClaim.assetId))
+
+  const amountCryptoPrecision = useMemo(() => {
+    return fromBaseUnit(activeClaim.amountCryptoBaseUnit, asset?.precision ?? 0, 8)
```

Comment:
q: should we keep full precision here and let display components handle trimming/rounding, or does it currently look ugly with full precision?

Reply:
Oop yeah, that was in place before updating to use the `Amount.Crypto` component. Will update.

Reply:
https://github.com/shapeshift/web/pull/7563/commits/c2d75468f60ec9899e9495a38d03f8017b559c7e